src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -207,4 +208,28 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('date')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('format')->defaultNull()->end()
+                        ->scalarNode('interval_format')->defaultNull()->end()
```

Comment:
IMO this should specify the same default as in https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Extension/Core.php#L18 since you also added the defaults for `number_format`